cookbooks/ambari/recipes/ambari_views_setup.rb

```diff
@@ -0,0 +1,40 @@
+#
+# Cookbook Name:: ambari
+# Recipe:: ambari_views_setup
+#
+# Copyright (c) 2016 The Authors, All Rights Reserved.
+
+
+http_request 'Polling Ambari WebUi' do
+  url "#{node['ambari']['ambari_server_base_url']}"
+  retries 24
+  retry_delay 5
+  # notifies :run, 'bash[create_instance_of_Files_view]', :immediately
+end
+
+# create an instance of Files View
+bash 'create_instance_of_Files_view' do
+  # action :nothing
+  code <<-EOH
+    curl --user #{node['ambari']['admin']['user']}:#{node['ambari']['admin']['password']} -i -H 'X-Requested-By: ambari' -X POST #{node['ambari']['ambari_views_url']}/FILES/versions/1.0.0/instances/FILES_NEW_INSTANCE \
+--data '{
+  "ViewInstanceInfo" : {
```

Comment:
Can we move this to an attribute? That way we add configuration for different views to the attribute and resource over here loops through the data structure and configures each view. Something similar to we do with [Kerberos](https://github.com/bloomberg/chef-bach/blob/master/cookbooks/bcpc-hadoop/attributes/kerberos.rb#L5-L22).